src/components/WeekTabs/WeekTabs.jsx

```diff
@@ -0,0 +1,79 @@
+import { React, useEffect, useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+
+import PropTypes from 'prop-types';
+
+import { TabsBackground,
+    TabsList,
+    DaysWrapper,
+    TabsListItem,
+    DayTitle,
+    FullDayTitle,
+    ShortDayTitle,
+    WeekData } from "./WeekTabsStyled";
+
+const WEEK_DAYS = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
+const WEEKDAY_VALUES = new Map();
```

Comment:
One more time, I dont see a reason for this map. Array is simpler but powerful enough

Reply:
I think you could use WEEK_DAYS instead